Compilation error of vvenc_unit_test of Visual Studio 2017

Hi!
See the two errors in compiling using VS2017 with Intel x64 target:

```
C:\Users\marti\Downloads\vvenc\test\vvenc_unit_test\vvenc_unit_test.cpp(840): error C2398: Element '1': conversion from 'const int' to 'unsigned int' requires a narrowing conversion [C:\Users\marti\Downloads\vvenc\build\test\vvenc_unit_test\vvenc_unit_test.vcxproj]
C:\Users\marti\Downloads\vvenc\test\vvenc_unit_test\vvenc_unit_test.cpp(841): error C2398: Element '1': conversion from 'const int' to 'unsigned int' requires a narrowing conversion [C:\Users\marti\Downloads\vvenc\build\test\vvenc_unit_test\vvenc_unit_test.vcxproj] 
```

Only the vvenc_unit_test target compiling caused two errors; solutions need to be converted.
But other tools of the toolchains were okay.
Thanks! :)

- Martin Eesmaa

--- a/test/vvenc_unit_test/vvenc_unit_test.cpp
+++ b/test/vvenc_unit_test/vvenc_unit_test.cpp
@@ -836,7 +836,7 @@ static bool test_AffineGradientSearch()
   unsigned num_cases = NUM_CASES;
   bool passed = true;
 
-  static constexpr int bd = 10;
+  static constexpr unsigned bd = 10;
   auto random_gen = InputGenerator<Pel>{ bd, /*is_signed=*/true };
   auto minmax_gen = MinMaxGenerator<Pel>{ bd, /*is_signed=*/true };
 

